src/pretty.js

```diff
@@ -135,6 +133,8 @@ function _renderToStringPretty(
 				!nodeName.prototype ||
 				typeof nodeName.prototype.render !== 'function'
 			) {
+				// let cctx = getContext(nodeName, context);
```

Comment:
Curious why this can be commented out as it#s supposedly passed in 150 . Was it an error in the code?

Reply:
Code was duplicated with the branch above, fixed now